pkg/dmsclient/dmsclient.go

```diff
@@ -103,14 +108,20 @@ func (c *dmsClient) newRequest(method, path string, body interface{}) (*http.Req
 }
 
 func (c *dmsClient) do(req *http.Request) (*http.Response, error) {
+	start := time.Now()
```

Comment:
I like this implementation a lot better, thank you.

Reply:
:+1: